tests/unit/api/test_user_defined_function.py

```diff
@@ -19,11 +19,7 @@
 @pytest.fixture(name="catalog")
 def catalog_fixture(snowflake_feature_store):
     """Catalog fixture"""
-    try:
-        yield Catalog.create(name="test_catalog", feature_store_name=snowflake_feature_store.name)
-    finally:
-        # change back to default catalog to avoid side effects on other tests
-        Catalog.activate("default")
```

Comment:
hm how come this isn't needed anymore?

Reply:
It's not longer needed since there is no longer a default catalog, and all tests now include the creation and activation of a catalog.